ex1/exceptions.py

```diff
@@ -46,3 +58,19 @@
 # the first by the second. This function should handle exceptions that might
 # occur print out what went wrong and return None if no results could be
 # computed.
+
+import sys
+
+def safe_divide(a,b):
+    """Divides a by b without interrupting when an exception is raised.
+    """
+    try:
+        res = a/b
+    except:
```

Comment:
In general you should not except any Exception since this will effectively hide any error. See https://github.com/TUW-GEO-python-intro/Exercise-1/pull/20/files#diff-e4b345aa8d30ac9c4f2351b3e80ff34fR101 on how to explicitly except and handle different types of Exceptions.